components/runtime/squeak4/Makefile

```diff
@@ -0,0 +1,72 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Package Makefile for the "Classic VM" Squeak Smalltalk V4 system
+# See http://squeak.org and http://squeakvm.org
+#
+# The squeak4 vm can only run Squeak4 images; it will not work with 
+# Squeak5 images; squeak5 uses  different technology called "Cog Spur"
+#
+# squeak4 uses cmake and a custom configure script (not GNU autoconf)
+# in order to build this package, "cmake" must be installed
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		squeak4
+COMPONENT_VERSION=	4.16.7
+COMPONENT_SUMMARY=	The Squeak V4 Smalltalk Virtual Machine
+COMPONENT_PROJECT_URL=	http://www.squeak.org
+COMPONENT_ARCHIVE_URL=  http://squeakvm.org
+COMPONENT_FMRI=		runtime/squeak4
```

Comment:
If it's a compiler, it should be developer/squeak4.

Reply:
It's a virtual machine, similar to JVM or DOT.NET runtime. Together with appropriate image triple it is a very capable environment. The compiler is just a part of it. So, either it should either have its own category or runtime would be the best fit.

Reply:
OK.